.pipelines/stages/jobs/steps/nuget-win-step.yml

```diff
@@ -16,7 +16,7 @@ steps:
     DisplayName: 'ESRP - Sign C# dlls'
 
 - powershell: |
-    $VERSION = '0.1.0-rc1'
+    $VERSION = '0.1.0-rc3'
```

Comment:
Can we read from VERSION_INFO file?

Reply:
The rc format for python and nuget is different. Unless we create a different version info file.

Reply:
Yeah, is it possible to find a common ground between the two?

Reply:
0.1.0-rc3 vs 0.1.0rc3 we would need to have different suffix, but python packaging directly read from a file.